booklore-ui/src/app/features/settings/local-settings/local-settings.component.scss

```
@@ -0,0 +1,115 @@
+@use "../../../shared/styles/settings-shared" as settings;
+
+.local-settings-container {
+  @include settings.settings-page-container;
+}
+
+@include settings.settings-page-header;
+
+.settings-title {
+  @include settings.settings-page-title;
+  margin-bottom: 0.75rem;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)